### PR TITLE
Proper Readme

### PR DESCRIPTION
**Pull request by dstathis**
Add a readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,66 @@
...
+
+The [Cassandra] operator provides a NoSQL distributed database solution. It is part of the Observability stack in the [Juju] charm [ecosystem]. Cassandra is highly scalable and fault tolerant.
+
+[Cassandra]: https://cassandra.apache.org/
+[Juju]: https://jaas.ai/
+[ecosystem]: https://charmhub.io/
+
+## Setup
+
+A typical setup using [snaps](https://snapcraft.io/), for deployments
```
Comment (balbirthomas) [inline README.md]: I thought @jlounder guidelines suggested that we do not add setup and build instructions in the readme. Please do check if I am mistaken.

**Review by balbirthomas — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1 +1,66 @@
...
+
+    charmcraft build
+
+## Usage
+
+Create a Juju model (say "lma" for your observability operators)
+
+    juju add-model lma
+
+### Deploy Alertmanager with PagerDuty configuration
```
Comment (jlounder) [inline README.md]: Un-edited copy-pasta

File: README.md (hunk 1 above)
Comment (jlounder) [reply README.md]: Correct. Please remove setup and build. These are generic instructions not specific to this charm. The main README should be for a general charm consumer. We can add further documentation with "contribution" instructions, which includes build steps, etc.

File: README.md [hunk 3]
```diff
@@ -1 +1,66 @@
...
+
+    juju deploy ./cassandra.charm --resource cassandra-image='dstathis/cassandra-operator-image:latest'
+
+### Scale Out Usage
+
+You may add additional Cassandra units for high availability
+
+    juju add-unit cassadnra
+
```
Comment (sed-i) [inline README.md:26]: Need "Relations" section?

File: README.md [hunk 4]
```diff
@@ -1 +1,66 @@
...
+## Usage
+
+Create a Juju model (say "lma" for your observability operators)
+
+    juju add-model lma
+
+### Deploy Alertmanager with PagerDuty configuration
+
+    juju deploy ./cassandra.charm --resource cassandra-image='dstathis/cassandra-operator-image:latest'
```
Comment (jlounder) [inline README.md:19]: Is there a reason we have to use your custom image, versus the appropriate upstream image?

**Review by sed-i — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (rbarry82) [reply README.md:26]: Also a typo

File: README.md (hunk 4 above)
Comment (dstathis) [reply README.md:19]: Yes. That image supports exporting metrics

**Pull request merged**